spdx-utils/src/main/kotlin/model/SpdxConstants.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+object SpdxConstants {
+    /**
+     * Represents a not present value, which has been determined to actually be not present. This representation must
+     * not be used if [NOASSERTION] could be used instead.
+     */
+    const val NONE = "NONE"
+
+    /**
+     * Represents a not present value where any of the following cases applies:
+     *
+     * 1. no attempt was made to determine the information.
+     * 2. intentionally no information is provided, whereas no meaning should be derived from the absence of the
+     *    information.
+     */
+    const val NOASSERTION = "NOASSERTION"
+
+    internal fun isNoneOrNoAssertion(value: String) = value in setOf(NONE, NOASSERTION)
```

Comment:
Maybe more generically name this `isNotPresent()`?

Reply:
I wonder whether we should turn this into an `enum class`. One advantage would be that `setOf(NONE, NOASSERTION)` here wouldn't need to be recreated on each function call, but simply `enumValues<>()` could be used.

Reply:
> Maybe more generically name this `isNotPresent()`?

I used that name before.